src/effects/native/phasereffect.cpp

```diff
@@ -115,13 +129,18 @@ void PhaserEffect::processChannel(const ChannelHandle& handle,
                                   const unsigned int sampleRate,
                                   const EffectProcessor::EnableState enableState,
                                   const GroupFeatureState& groupFeatures) {
-
     Q_UNUSED(handle);
     Q_UNUSED(enableState);
-    Q_UNUSED(groupFeatures);
-    Q_UNUSED(sampleRate);
 
-    CSAMPLE frequency = m_pLFOFrequencyParameter->value();
+    CSAMPLE period = std::max(m_pLFOPeriodParameter->value(), 0.05);
+    if (m_pSyncParameter->toBool() && groupFeatures.has_beat_length) {
+        // Period is a number of beats
+        period = period * groupFeatures.beat_length;
+    } else {
+        // Period is a number of seconds
+        // TODO: remove assumption of stereo signal
+        period = period * sampleRate * 2;
```

Comment:
My tracks have a much higher average BPM. Did you consider to pick 120 bpm to have similar scales? 
It looks like this decision is a common one among all beat synced effects. 
Can we define this somewhere central? 

How about pick a reasonable clock in this case, like the other deck bpm or the master clock? 
This is probably out of the focus of this PR.